public/app/features/explore/RichHistory/RichHistoryCard.tsx

```diff
@@ -215,7 +216,11 @@ export function RichHistoryCard(props: Props) {
       .join('\n');
 
     copyStringToClipboard(queriesText);
-    dispatch(notifyApp(createSuccessNotification('Query copied to clipboard')));
+    dispatch(
+      notifyApp(
+        createSuccessNotification(t('explore.history.rich-history-card.query-copied', 'Query copied to clipboard'))
```

Comment:
According to [the guidelines](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#phrase-id-naming-convention) we should use up to 3 segments in keys.

```suggestion
        createSuccessNotification(t('explore.rich-history-notification.query-copied', 'Query copied to clipboard'))
```